### PR TITLE
AHIT: Update Setup Guide

### PR DESCRIPTION
**Pull request by CookieCat45**
## What is this fixing or adding?
Recently as per my own request, Gears for Breakfast added a new Steam beta branch for A Hat in Time named `tcplink` (which was very nice of them!) This branch was created specifically for the Archipelago mod to function on, and is a different version than the one that was downloaded via the troublesome `download_depot` command that seems to be more stable. I have updated the setup guide accordingly to accomodate for this new beta branch. It is now much more straightforward and less error prone.

Screenshot 2024-07-16 144145

## How was this tested?
I downloaded the `tcplink` branch myself and verified that the mod functions correctly with it.

### CODE REVIEW

**Review by remyjette — APPROVED** [review]
Wow, this is so much easier!

Tested, worked for me. Docs are clear.

Huge kudos to the devs, "This branch was created specifically for the Archipelago mod" you love to see it

**Review by agilbert1412 — APPROVED** [review]

File: worlds/ahit/docs/setup_en.md [hunk 1]
```diff
@@ -12,41 +12,29 @@
 
 ## Instructions
 
-1. Have Steam running. Open the Steam console with this link: [steam://open/console](steam://open/console)   
-This may not work for some browsers. If that's the case, and you're on Windows, open the Run dialog using Win+R,
-paste the link into the box, and hit Enter.
+1. **BACK UP YOUR SAVE FILES IN YOUR MAIN INSTALL IF YOU CARE ABOUT THEM!!!**   
+Go to `steamapps/common/HatinTime/HatinTimeGame/SaveData/` and copy everything inside that folder over to a safe place.
+**This is important! Changing the game version CAN and WILL break your existing save files!!!**
```
Comment (black-sliver) [inline worlds/ahit/docs/setup_en.md:17]:
```suggestion
1. **BACK UP YOUR SAVE FILES IN YOUR MAIN INSTALL IF YOU CARE ABOUT THEM!!!**  
   Go to `steamapps/common/HatinTime/HatinTimeGame/SaveData/` and copy everything inside that folder over to a safe place.
   **This is important! Changing the game version CAN and WILL break your existing save files!!!**
```
Indentation as per markdown style guide, also there was 1 extra space at the end of the first line?

File: worlds/ahit/docs/setup_en.md [hunk 2]
```diff
@@ -12,41 +12,29 @@
...
-`download_depot 253230 253232 7770543545116491859`. ***Wait for the console to say the download is finished!***
-This can take a while to finish (30+ minutes) depending on your connection speed, so please be patient. Additionally,
-**try to prevent your connection from being interrupted or slowed while Steam is downloading the depot,**
-or else the download may potentially become corrupted (see first FAQ issue below).
+2. In your Steam library, right-click on **A Hat in Time** in the list of games and click on **Properties**.
 
 
-3. Once the download finishes, go to `steamapps/content/app_253230` in Steam's program folder.
+3. Click the **Betas** tab. In the **Beta Participation** dropdown, select `tcplink`.   
+While it downloads, you can subscribe to the [Archipelago workshop mod.]((https://steamcommunity.com/sharedfiles/filedetails/?id=3026842601))
```
Comment (black-sliver) [inline worlds/ahit/docs/setup_en.md:24]:
```suggestion
3. Click the **Betas** tab. In the **Beta Participation** dropdown, select `tcplink`.  
   While it downloads, you can subscribe to the [Archipelago workshop mod.]((https://steamcommunity.com/sharedfiles/filedetails/?id=3026842601))
```
As above

File: worlds/ahit/docs/setup_en.md [hunk 3]
```diff
@@ -12,41 +12,29 @@
...
 
 
-4. There should be a folder named `depot_253232`. Rename it to HatinTime_AP and move it to your `steamapps/common` folder.
+4. Once the game finishes downloading, start it up.   
+In Game Settings, make sure **Enable Developer Console** is checked.
```
Comment (black-sliver) [inline worlds/ahit/docs/setup_en.md:28]:
```suggestion
4. Once the game finishes downloading, start it up.  
   In Game Settings, make sure **Enable Developer Console** is checked.
```
As above.

**Review by black-sliver — COMMENTED** [review]
Style guide suggests to have `\` for the forced line break, but if you use `  `, I think it should only be 2 spaces?

If you want to follow our style guide to the letter, use `\` at the end of lines that should force breaks., however mixing `\` and `  ` is the worst, so that'd mean going over the file and replacing all instances, which is why I didn't put that into my suggestions.

**Review by black-sliver — APPROVED** [review]
Style concerns have been addressed, actual instructions were tested by peer review

**Pull request merged**